### DISCUSSION TITLE
Absolute paths using $get?

### DISCUSSION THREAD

**Discussion by OvalMedia** [filamentphp/filament#4843] [Help] [answered]
I have an `invoice` 1:n `invoiceitems`

In `afterStateUpdated` of a `TextInput` of an invoiceitem I can use the `$get` closure to access other form fields. And by using a relative path I can access fields of the invoice from an invoiceitem (e.g. `$get('../../date')`. 

I have a couple of fields in the invoice itself as well as in the items which all should trigger the same function (via `afterStateUpdated`) to recalculate everything. But using relative paths with $get would not work depending on the context.

E.g. `$get('../../date')` will work in an invoiceitem's field but not in an invoice's field where I would have to use `$get('date')`.

I guess I have 2 questions:

1. Is there a way to use absolute paths in $get? I already tried `$get('/date')` but that did not work.
2. Can I access the root form (invoice) no matter what the context?

**Comment by danharrin** [comment, marked as answer]
The second param of `$get` is `isAbsolute: true`, which you could use. But beware, this isn't "absolute" from the current form, it is absolute for the entire Livewire component. So you'd still need to prefix whichever state path you're trying to access (usually `data.` or `mountedActionData.`)
